jlh/JonFirstApp/app/src/main/java/com/e_conomic/jonfirstapp/DisplayMessageFragment.java

```diff
@@ -25,6 +31,12 @@
     public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setRetainInstance(true);
+
+        String no_prev_message = getResources().getString(R.string.no_prev_message);
```

Comment:
Wondering: didn't you say that values created "on the go" should be `snake_case`? Or am I remembering it incorrectly / misunderstanding? :)